cmake/modules/boards.cmake

```diff
@@ -51,6 +51,13 @@ include(extensions)
 # If user tries to change the BOARD, the BOARD value is reset to the BOARD_CACHED value.
 zephyr_check_cache(BOARD REQUIRED)
 
+if(SYSBUILD)
```

Comment:
Please remove
```suggestion
```

we should not do specific check for sysbuild everywhere in the build system.

The handling of sysbuild invoked builds is handled through the `zephyr_get(...)` function which takes a `SYSBUILD` flag indicating if sysbuild handled settings should be considered.

Reply:
Fixed